src/pipecat/transports/network/websocket_server.py

```diff
@@ -186,6 +192,12 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
             await self._write_frame(frame)
             self._next_send_time = 0
 
+    async def send_message(self, frame: TransportMessageFrame | TransportMessageUrgentFrame):
+        message_frame = TextFrame(
+            text=json.dumps(frame.message),
+        )
+        await self._write_frame(message_frame)
```

Comment:
I don't think this should go in here. This should be the responsibility of the serializer that knows how to transform a `TransportMessageFrame` into whatever is needed.

So, in the demo you would implement a serializer that performs the conversion.